CurrencyConverterService/RatesRefreshing.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.EntityFrameworkCore;
+
+namespace CurrencyConverterService.Models
+{
+    class RatesRefreshing
+    {
+        Context dataBase;
+        Logging logging;
+        public RatesRefreshing()
+        {
+            dataBase = new Context();
+            this.logging = new Logging();
+        }
+        public void AddRates(List<Rates> rates)
+        {
+            try
+            {
+                foreach (var rate in rates)
+                {
+                    dataBase.Rates.Add(rate);
+                }
+                dataBase.SaveChanges();
+                logging.AddInformation("Rates table updated");
+            }
+            catch (Exception exception)
+            {
+                logging.AddError(exception.ToString());
```

Comment:
I'm not sure, But do you think this log record is helpful? Do we know where exception is happened from this log record?